__tests__/pet.test.js

```diff
@@ -0,0 +1,153 @@
+const Pet = require('../src/pet');
+
+describe('constructor', () => {
+  it('returns an object', () => {
+    expect(new Pet('Fido')).toBeInstanceOf(Object);
+  });
+
+  it('sets the name property', () => {
+    const pet = new Pet('Fido');
+    expect(pet.name).toEqual('Fido');
+  });
+
+  it('has a initial age of 0', () => {
+    const pet = new Pet('Fido');
+    expect(pet.age).toEqual(0);
+    expect(pet.hunger).toEqual(0);
+    expect(pet.fitness).toEqual(10);
+
+  });
+
+});
+
+describe('growUp', () => {
+  it('increments the age by 1', () => {
+    const pet = new Pet('Fido');
+
+    pet.growUp();
+
+    expect(pet.age).toEqual(1);
+    expect(pet.hunger).toEqual(5);
+    expect(pet.fitness).toEqual(7);
+
+    pet.growUp();
+
+    expect(pet.age).toEqual(2);
+    expect(pet.hunger).toEqual(10);
+    expect(pet.fitness).toEqual(4);
+
+  });
+});
+
+describe('walk', () => {
+  it('increments the fitness by 4, but max 10', () => {
```

Comment:
Building on the idea of constants being defined centrally, if any of them were to change in the future this would make the name of this test wrong. Rather than referring to specific values in the test definition like this, it could be named something like 'increase fitness by proper increment without exceeding limit'